app/src/main/java/org/oppia/app/topic/revisioncard/ReturnToTopicClickListener.kt

```diff
@@ -0,0 +1,5 @@
+package org.oppia.app.topic.revisioncard
+
+interface ReturnToTopicClickListener {
```

Comment:
Add Kdoc for this interface. You can check other interface to know how we do that.

Reply:
Done